File: aulas/aula15_c1/jokenpo/joke.py
#!/usr/bin/env python
#r rock, p paper, s scissors

op1 = raw_input()
op2 = raw_input()

if op1 == op2:
    print "empate"
elif (op1 == "r" and op2 == "s") or (op1 == "s" and op2 == "p") or \
    (op1 == "p" and op2 == "r"):
    print "jog1"
else:
    print "jog2"

File: aulas/aula15_c1/jokenpo/joke.c
#include <stdio.h>

int main(){ char op1 = 0; char op2 = 0;
    printf("Digite as duas opcoes\n");
    scanf("%c %c", &op1, &op2);

    if (op1 == op2)
    {
        printf("empate\n");
    }
    else
    {
        if((op1 == 'r' && op2 == 's') ||
             (op1 == 's' && op2 == 'p') ||
             (op1 == 'p' && op2 == 'r'))
        {
            printf("jog1\n"); } else {
            printf("jog2\n");
        }
    }

    return 0;
}
